src/styles/blocks/reviews.scss

```diff
@@ -0,0 +1,11 @@
+.reviews {
+  display: flex;
+  margin-left: 37px;
+  margin-right: 0;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the values for margins should be combined into a single property if they are not all unique. In this case, since `margin-right` is `0`, you can combine `margin-left` and `margin-right` into one `margin` shorthand property.